src/components/OrganizationDashCards/CardItem.module.css

```diff
@@ -2,7 +2,10 @@
   position: relative;
   display: flex;
   align-items: center;
-  padding: 0.75rem 0;
+  padding: 1rem 3rem 1rem 1.5rem;
+  border: 1px solid #dddddd;
```

Comment:
use css variable here

Reply:
@rishav-jha-mech can you give me some idea about how to use the CSS variables ?

Reply:
like this,
```css
border: 1px solid var(--bs-gray-200);
```
`--bs-gray-200` this variable is from bootstrap

Reply:
@rishav-jha-mech, can I write inline bootstrap CSS variables in the classname? Is that ok to write ??